WTWSMS/localisation/AO_events.csv

```diff
@@ -187,7 +187,7 @@ ruthless_desc;This character believes that the ends justify the means, and will
 honorable;Honorable;Honorable;;;;;;;;;;;;x
 honorable_desc;This character is a firm believer in the importance of duty, mercy, and honor.;Ce personnage croit fermement en l'importance du devoir, de la piti� et de l'honneur.;;;;;;;;;;;;x
 plot_kidnap_character_title;Kidnap [This.GetTitledFirstName];Kidnapper [This.GetTitledFirstName];[This.GetTitledFirstName] entf�hren;;;;;;;;;;;x
-plot_kidnap_character_short;Kidnap [This.GetTitledFirstName];Kidnapper [This.GetTitledFirstName];;;;;;;;;;;;x
+plot_kidnap_character_short;Plot to Kidnap;Kidnapper [This.GetTitledFirstName];;;;;;;;;;;;x
```

Comment:
The title works, but not the short?

Reply:
Title is what appears in the plot screen?

Reply:
Which one is it that works?

Reply:
_short is for targetted decision. I believe _short and _title have different usage

Reply:
Then it should be ROOT no?

Reply:
Why Root?

Reply:
Because ROOT is the the targeted character in targeted decisions.

Reply:
That could work then, I will test.

Reply:
It still returned the decision taker...

Reply:
If the scopes are different in plots use FROM.

Reply:
From didn't work too.

Reply:
What worked then? Those are the only scopes, all other plots use them...

Reply:
I couldn't find a working scope, so I used "Plot to Kidnap" instead.